src/stores/room-list/RoomListStore2.ts

```diff
@@ -91,29 +92,37 @@ export class RoomListStore2 extends AsyncStore<ActionPayload> {
         await this.updateAlgorithmInstances();
     }
 
-    private onRVSUpdate = () => {
+    private onRVSUpdate = async (quiet = false) => {
```

Comment:
I'd prefer an object here, so we can see `quiet: true` at the call site for readability.

I would slightly prefer also having `trigger = true` instead of `quiet = false` (inverting the sense of it) as an option to avoid the double negative of "false quiet" means "trigger".

When is `quiet` appropriate to use...? A function doc might help a lot here.

I see the `AsyncStore` uses the bare boolean though... so maybe just adding a quick doc is enough for now.